exporter/test_excel.py

```diff
@@ -0,0 +1,79 @@
+from exporter.excel import _clean, export
+import openpyxl
+import unittest
+
+
+class Test_Clean(unittest.TestCase):
+
+    def setUp(self):
+        # Data to export
+        self.data = []
+        self.fields = []
+
+        for i in range(0, 10):
+            self.data.append('%i' %i)
+            self.fields.append('id%i' %i)
+
+        self.test_dict = dict(zip(self.fields, self.data))
+        self.test_list = self.data
+        self.test_set = sorted(set(self.data))
+        self.test_tuple = tuple(self.data)
+
+    def test_clean_dict(self):
+        value = _clean(self.test_dict)
+
+    def test_clean_list(self):
+        value = _clean(self.test_list)
+        self.assertEqual(value, '0, 1, 2, 3, 4, 5, 6, 7, 8, 9')
+
+    def test_clean_set(self):
+        value = _clean(self.test_set)
+        self.assertEqual(value, '0, 1, 2, 3, 4, 5, 6, 7, 8, 9')
+
+    def test_clean_tuple(self):
+        value = _clean(self.test_tuple)
+        self.assertEqual(value, '0, 1, 2, 3, 4, 5, 6, 7, 8, 9')
+
+
+class TestExport(unittest.TestCase):
+
+    def setUp(self):
+        # Data to export
+        self.data = []
+        self.fields = []
+
+        for i in range(0, 10):
+            self.data.append('%i' %i)
+            self.fields.append('id%i' %i)
+
+        self.dict_data = dict(zip(self.fields, self.data))
+
+    def test_get_excel_file(self):
+        f = export(self.dict_data, self.fields, 'Test')
+        wb = openpyxl.reader.excel.load_workbook(f)
+        try:
+            ws = wb.get_sheet_by_name('Test')
+        except KeyError:
+            ws = None
+
+        self.assertTrue(ws)
```

Comment:
From your test below, it seems `wb.get_sheet_by_name('Test')` does not return True when it succeeds. I don't know the API exactly, but it seems to me you will get some kind of Excel Sheet object.

My question to you is: what is it you're testing here? If you're testing whether an object is returned, you could [`assertIsNotNone`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsNotNone) on the returned object and maybe test for expected output.

You could maybe even combine it with the test below, as there seems to be some overlap, but that's up to you. :)